### PR TITLE
F/delete wildcard file

### PR DESCRIPTION
**Pull request by juhaszviktor**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build? (admin please type: ok to test)

**Comment by furiel** [comment]
@kira-syslogng ok to test

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build? (admin please type: ok to test)

**Comment by MrAnno** [comment]
@kira-syslogng ok to test

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
success

File: modules/affile/tests/test_wildcard_file_reader.c [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+{
+  return;
+}
+
+void file_reader_notify_method(LogPipe *s, gint notify_code, gpointer user_data)
+{
+  return;
+}
+
+int iv_task_registered(const struct iv_task *_t)
```
Comment (furiel) [inline modules/affile/tests/test_wildcard_file_reader.c]: Just as a reminder, this will not compile with below libivykis 0.38.

**Comment by pzoleex** [comment]
@kira-syslogng test this please test branch=pzolee-wildcard-file-source-removed-max-files;

**Comment by pzoleex** [comment]
@kira-syslogng do perftest

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
Performance test: Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
failure

**Comment by kira-syslogng** [comment]
success

File: modules/affile/tests/test_wildcard_file_reader.c (hunk 1 above)
Comment (juhaszviktor) [reply modules/affile/tests/test_wildcard_file_reader.c]: I've fixed it

**Comment by pzoleex** [comment]
@kira-syslogng test this please test branch=pzolee-wildcard-file-source-removed-max-files;

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
success

**Comment by kira-syslogng** [comment]
success

**Comment by lbudai** [comment]
@juhaszviktor : thanks!

**Comment by juhaszviktor** [comment]
@lbudai : Welcome! :)

**Pull request merged**